### PR TITLE
 add nn.functional.normalize API for API2.0, fix L1Loss API typo

### PR DESCRIPTION
**Pull request by LutaoChu**
### PR types

New features,Bug fixes
### PR changes

APIs,Docs
### Describe

1 add nn.functional.normalize API for API2.0,
2 fix L1Loss API typo: x -> input, remove 'scalar'

### CODE REVIEW

File: python/paddle/nn/functional/norm.py [hunk 1]
```diff
@@ -24,5 +29,84 @@
...
+    where, $\sum_i{x_i^p}$ is calculated along the `axis` dimension.
+
+
+    Args:
+        x (Tensor): The input tensor could be N-D tensor, and the input data type could be float32 or float64.
+        p (float|int, optional): The exponent value in the norm formulation. Default: 2
+        axis (int, optional): The axis on which to apply normalization. If `x` is 1-D tensor, `axis` is fixed to 0. If `axis < 0`, \
+            the dimension to normalization is rank(X) + axis. -1 is the last dimension.
+        epsilon (float, optional): Small float added to denominator to avoid dividing by zero. Default is 1e-12.
+	    name (str, optional): Name for the operation (optional, default is None). For more information, please refer to :ref:`api_guide_Name`.
```
Comment (wawltor) [inline python/paddle/nn/functional/norm.py]: name 对齐

File: python/paddle/nn/functional/norm.py [hunk 2]
```diff
@@ -24,5 +29,84 @@
...
+    Returns:
+        Tensor, the output has the same shape and data type with `x`.
+
+    Examples:
+
+        .. code-block:: python
+
+            import paddle
+            import paddle.nn.functional as F
+            import numpy as np
```
Comment (wawltor) [inline python/paddle/nn/functional/norm.py]: import numpy as np 放在第一行import， pep8规范

File: python/paddle/fluid/tests/unittests/test_normalize.py [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+        y = F.normalize(x, p=1.5)
+        self.assertTrue(np.allclose(y.numpy(), self.expected1))
+
+        y = F.normalize(x, axis=0)
+        self.assertTrue(np.allclose(y.numpy(), self.expected2))
+
+        x = paddle.to_variable(self.input_np2)
+        y = F.normalize(x)
+        self.assertTrue(np.allclose(y.numpy(), self.expected3))
+
```
Comment (wawltor) [inline python/paddle/fluid/tests/unittests/test_normalize.py:57]: eps单测是否需要增加一个？

File: python/paddle/nn/functional/norm.py (hunk 1 above)
Comment (LutaoChu) [reply python/paddle/nn/functional/norm.py]: done

File: python/paddle/nn/functional/norm.py (hunk 2 above)
Comment (LutaoChu) [reply python/paddle/nn/functional/norm.py]: done. 是因为按字母序吗？

File: python/paddle/fluid/tests/unittests/test_normalize.py (hunk 3 above)
Comment (LutaoChu) [reply python/paddle/fluid/tests/unittests/test_normalize.py:57]: eps用来防止除零，上面这行就是验证除零时eps是否生效了，输入特意构造了全零tensor

**Review by TCChenlong — APPROVED** [review]
LGTM

**Review by wawltor — APPROVED** [review]
LGTM

**Review by WuHaobo — APPROVED** [review]
LGTM
